src/Controllers/common.ts

```diff
@@ -147,28 +147,16 @@ export async function handleCreateByName(
 
   // Check if link object exsits
 
-  try {
```

Comment:
These errors will be caught by our error handling, right?

Reply:
Yes, it will be caught by our errorhandler.